com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -1799,10 +1838,24 @@ static public EOSSingleton Instance
         /// </summary>
         void Awake()
         {
-            if (InitializeOnAwake)
+            // If the backing field s_instance already exists, and it has indicated it is running,
+            // then this EOSManager shouldn't perform any operations.
+            // So at this time if that condition is met, disable this behavior so that it doesn't run Update, etc
+            if (isThereAnExistingRunningManager)
```

Comment:
What does the lifecycle of objects look like for the scenario where the manager is not null and is not running. Is there a way to make the existing non-running instance running?

Reply:
So I think now that I've gotten rid of the "initialize on awake", this isn't really possible.

Before, you could have a EOSManager that is alive, but marked to not initialize things. However, that's pretty silly.

Reply:
Agreed on the silliness 😆 

Sounds good, and makes sense to me in context.